packages/web-app-files/tests/unit/components/FilesList/Pagination.spec.js

```diff
@@ -0,0 +1,108 @@
+import Vuex from 'vuex'
+import GetTextPlugin from 'vue-gettext'
+import DesignSystem from 'owncloud-design-system'
+import Pagination from '../../../../src/components/FilesList/Pagination.vue'
+import { createLocalVue, mount, RouterLinkStub, shallowMount } from '@vue/test-utils'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.use(DesignSystem)
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+
+describe('Pagination', () => {
+  describe('pages', () => {
+    describe('when value is less than or equals one', () => {
+      it.each([-1, 0, 1])('should not show wrapper', pages => {
+        const store = createStore(0, pages)
+        const wrapper = getWrapper(store)
+
+        expect(wrapper.find(selectors.filesPagination).exists()).toBeFalsy()
+      })
+    })
+
+    describe('when value is greater than one', () => {
+      const wrapper = getWrapper(createStore(1, 2))
+
+      it('should show wrapper', () => {
+        const paginationEl = wrapper.find('.files-pagination')
+
+        expect(paginationEl.exists()).toBeTruthy()
+        expect(paginationEl.attributes().pages).toBe('2')
+      })
+
+      it('should set provided current page', () => {
+        const paginationEl = wrapper.find(selectors.filesPagination)
+
+        expect(paginationEl.attributes().currentpage).toBe('1')
+      })
+    })
+  })
+
+  describe('current route', () => {
+    it('should use provided route to render pages', () => {
+      const wrapper = getMountedWrapper(createStore())
```

Comment:
will `getWrapper()` also work here?

Reply:
I tried a lot, but using shallow mount `currentRoute` props is always `[object Object]`. cannot assert its value using shallow mount here.